rpcserver.go

```diff
@@ -1874,18 +1966,14 @@ func (r *rpcServer) ListInvoices(ctx context.Context,
 		rHash := sha256.Sum256(paymentPreimge)
 
 		invoice := &lnrpc.Invoice{
-			Memo:         string(dbInvoice.Memo[:]),
-			Receipt:      dbInvoice.Receipt[:],
-			RHash:        rHash[:],
-			RPreimage:    paymentPreimge,
-			Value:        int64(invoiceAmount),
-			Settled:      dbInvoice.Terms.Settled,
-			CreationDate: dbInvoice.CreationDate.Unix(),
-			PaymentRequest: zpay32.Encode(&zpay32.PaymentRequest{
-				Destination: r.server.identityPriv.PubKey(),
-				PaymentHash: sha256.Sum256(paymentPreimge),
-				Amount:      invoiceAmount,
-			}),
+			Memo:           string(dbInvoice.Memo[:]),
+			Receipt:        dbInvoice.Receipt[:],
+			RHash:          rHash[:],
+			RPreimage:      paymentPreimge,
+			Value:          int64(invoiceAmount),
+			Settled:        dbInvoice.Terms.Settled,
```

Comment:
In the instances where an invoice is returned from the database, the following fields are never populated:
  * `FallbackAddr`
  * `Expiry` 
  *  `DescriptionHash`

Reply:
Right... Added :)